gql_ws_link/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: gql_ws_link
+version: 0.1.0
+homepage: https://github.com/gql-dart/gql/tree/master/gql_ws_link
+description: GQL Websocket Link
+environment: 
+  sdk: '>=2.7.2 <3.0.0'
+dependencies: 
+  meta: ^1.1.8
+  gql_exec: ^0.2.4
```

Comment:
It's best to require all `gql*` packages by relative path

Reply:
I think that's not true. Pub does not allow to publish with relative deps.

Reply:
oh right, my bad!